eftest/src/eftest/runner.clj

```diff
@@ -44,22 +44,22 @@
 (defn- ^Callable bound-callback [f]
   (cast Callable (bound-fn* f)))
 
-(defn- ^ExecutorService threadpool-executor []
-  (Executors/newFixedThreadPool (+ 2 (.availableProcessors (Runtime/getRuntime)))))
-
-(defn- pcalls* [fs]
-  (let [executor (threadpool-executor)]
-    (try
-      (->> fs
-           (map #(.submit executor (bound-callback %)))
-           (doall)
-           (map #(.get %))
-           (doall))
-      (finally
-        (.shutdownNow executor)))))
-
-(defn- pmap* [f xs]
-  (pcalls* (map (fn [x] #(f x)) xs)))
+(defn- ^ExecutorService threadpool-executor
+  [{:keys [threads] :or {threads :processors}}]
```

Comment:
Lets remove the `:processors` option, and instead set the default to 2 + available-processors when the number of threads is not set.